### PR TITLE
Config parameter "tls_disable_client_certs" is wrongly evaluated.

### PR DESCRIPTION
**Pull request by BharathB23**
Config parameter "tls_disable_client_certs" is evaluated wrongly, i.e. irrespective of the value set the action taken is that of when set to true.

### CODE REVIEW

File: command/server/listener.go [hunk 1]
```diff
@@ -177,7 +177,12 @@ PASSPHRASECORRECT:
 		if disableClientCerts && requireVerifyCerts {
 			return nil, nil, nil, fmt.Errorf("'tls_disable_client_certs' and 'tls_require_and_verify_client_cert' are mutually exclusive")
 		}
-		tlsConf.ClientAuth = tls.NoClientCert
+		if disableClientCerts {
+			tlsConf.ClientAuth = tls.NoClientCert
+		}
+		if !disableClientCerts {
```
Comment (jefferai) [inline command/server/listener.go]: This `if` section shouldn't be here. Only the first is needed.

Comment (BharathB23) [reply command/server/listener.go]: If "tls_disable_client_certs" is set to false, then if client provides a certificate shouldn't it be validated.

Comment (BharathB23) [reply command/server/listener.go]: Sorry I overlooked the default value set.

**Comment by jefferai** [comment]
Thanks!

**Pull request merged**